### ISSUE TITLE
Spaces in Pasted Text Interpreted as Tabs

### ISSUE THREAD

**Issue by Aljbab14** [TexteaInc/funix#61]
I am writing to bring to your attention an issue that I am currently experiencing while attempting to paste text into Funix. It appears that any spaces within the text are being interpreted as tabs, causing the cursor to jump to the next column. I would greatly appreciate any guidance or assistance you could provide in resolving this matter.

![image](https://user-images.githubusercontent.com/98681495/222633675-11d49834-eb06-476e-ba33-bcc63a1ef98e.png)

![image](https://user-images.githubusercontent.com/98681495/222633701-ae44b932-aeea-47d7-bee3-97cc5b021bc5.png)

**Comment by Yazawazi** [comment]
At: https://github.com/TexteaInc/funix/blob/00cee981c31db6482b580f9af07b5c3ed4629170/frontend/src/components/FunixFunction/ObjectFieldExtendedTemplate.tsx#L639

I will fix it later!

**Comment by Yazawazi** [comment]
Done in https://github.com/TexteaInc/funix/commit/90c3d47ddd14673cc9bb39f314daef910e364ca6, please check it!

I just tested Google Sheets and Microsoft Excel and they both seem to split columns with tabs.

![Image](https://user-images.githubusercontent.com/47273265/222672559-91e1c839-2397-4ba9-9da0-27eab3723afe.png)
![Image](https://user-images.githubusercontent.com/47273265/222672616-c299d451-1cca-4c7a-aaaf-c5c800b0b38d.png)

The code is:
```python
from funix import funix
from typing import List
import matplotlib.pyplot as plt
from matplotlib.figure import Figure


@funix(
    widgets={
        ("phrase", "year", "freq", "test"): "sheet"
    }
)
def sheet_trim(phrase: List[str], year: List[int], freq: List[int], test: List[str]) -> Figure:
    fig = plt.figure()
    plt.plot(year, freq)
    return fig
```

**Comment by forrestbao** [comment]
Thanks @Yazawazi . I think we should keep the behavior consistent with Goolge Docs and Microsoft Excel.

**Comment by Yazawazi** [comment]
Fixed in `0.3.7`
